py/preprocess_data.py

```diff
@@ -38,6 +41,15 @@
 # )
 # Merge to get question text.
 responses2 = responses2.merge(questions, on="question_id")
+# merge to get val_order
+responses2 = responses2.merge(xtab_order, on=["xtab1_var", "xtab1_val"], how="left")
```

Comment:
since `responses2` gets overwritten with merges, can we either keep overwriting `responses` or increment the number each time it's merged? I think `responses` throughout is simplest